app/src/views/HostApplicationTracker.tsx

```diff
@@ -26,5 +28,12 @@ function reducer(
 
 export function HostApplicationTracker(props: HostApplicationTrackerProps) {
   const [state, dispatch] = React.useReducer(reducer, initialState);
-  return <h1>Host application tracker</h1>;
+  return (
+    <Box>
+      <AuthenticatedHeader onClick={() => null} />
```

Comment:
It is unusual to define a function that always returns `null`. You can be more direct (and create marginally more efficient code) by setting this to be undefined. 

You can make the onClick handler optional by updating:

```diff
interface OwnProps {
-  onClick: () => void
+ onClick?: () => void
```

And handling the unassigned case within `AuthenticatedHeader`. I don't think we should render the hamburger menu if there is no `onClick` handler because that means that there is no way for that menu to open (so rendering it is misleading). 

You can do this, for example, to remove the menu while maintaining the current styling within `AuthenticatedHeader`:

```diff
        <Toolbar sx={{justifyContent: {xs: 'space-between'}}}>
+        {onClick ? (
          <IconButton
            aria-label="open drawer"
            edge="start"
            onClick={onClick}
            sx={{mr: 2, height: '40px', width: '40px', display: {md: 'none'}}}
          >
            <MenuIcon />
          </IconButton>
+         ): (
+          // This empty element acts as a spacer when IconButton is not rendered
+         <div style={{ width: '40px'}}></div>
+      )}
          <StyledLogo src={logo} alt="Home Unite Us logo" />
          <Stack direction="row" gap={1} sx={{alignItems: 'center'}}>
            <AvatarDropdownMenu />
          </Stack>
        </Toolbar>
```

Reply:
The hamburger menu is supposed to render the side nav of the page it's used on, but since those route's content haven't been made yet then I just set the onclick to null as a placeholder until the page is built.

Reply:
@Joshua-Douglas  Made the changes requested along with merge conflicts from taking so long 😅. Is this okay now to merge into main with these changes?